theseus_gui/src/pages/Browse.vue

```diff
@@ -394,11 +386,10 @@ async function clearFilters() {
   for (const facet of [...orFacets.value]) {
     await toggleOrFacet(facet, true)
   }
-
   onlyOpenSource.value = false
   selectedVersions.value = []
   selectedEnvironments.value = []
-  await onSearchChange(1)
+  await onSearchChangeToTop(1)
```

Comment:
why were these and others changed to to top? these operations should not affect the scroll pos